frontend/src/app/models/grammar-courses.ts

```diff
@@ -0,0 +1,9 @@
+export interface GrammarCourses {
+  id: number;
+  name: string;
+  assignment: string;
+  image: string;
+  availableUnits: number;
+  description: string;
+  status: string;
```

Comment:
`status: 'Fertig' | 'In Bearbeitung' | 'Offen'` dann kann man nur die 3 strings da rein schreiben :)